src/util/address.rs

```diff
@@ -81,7 +84,8 @@ impl fmt::Display for Error {
             Error::EmptyBech32Payload => write!(f, "the bech32 payload was empty"),
             Error::InvalidBech32Variant => write!(f, "invalid bech32 checksum variant"),
             Error::InvalidWitnessVersion(v) => write!(f, "invalid witness script version: {}", v),
-            Error::UnparsableWitnessVersion => f.write_str("Incorrect format of a witness version byte"),
+            Error::UnparsableWitnessVersion(ref e) => write!(f, "Incorrect format of a witness version byte: {}", e),
```

Comment:
This error occurs when parsing a string, idk if referring to a byte is really the right thing to do. Super minor nit though.

Reply:
Fixed in 6848ea43aa53e903323408bfe78bec342dc9efa4